skyportal/tests/flaky/test_frontend_flaky.py

```diff
@@ -355,3 +355,62 @@ def test_gcnevents_observations(
     driver.wait_for_xpath('//*[text()="ztfr"]')
     driver.wait_for_xpath('//*[text()="1.57415"]')
     driver.wait_for_xpath('//*[text()="20.40705"]')
+
+
+# @pytest.mark.flaky(reruns=2)
```

Comment:
I'm worried that we are putting all the frontend tests directly into the flaky category. Also there's a commented line here that should probably not be.

Reply:
I uncommented it. I am mostly worried about the tradeoff here between something breaking and these elements interrupting others. The less complicated ones I am still writing into frontend/